.github/templates/main.yml

```diff
@@ -104,6 +115,15 @@ run: |
 shell: bash
 #@ end
 
+#@ def buildNetFrameworkTest(relSolutionPath, **properties):
```

Comment:
Can we rename this to just `msbuild` and reuse it on line 78 (will likely need to move other stuff as parameters)?